src/main/java/teammates/common/util/LogHelper.java

```diff
@@ -0,0 +1,195 @@
+    private LogQuery query;
+    private Long endTime;
+    private List<String> versionList;
+    
+    public LogHelper() {
+        query = LogQuery.Builder.withDefaults();
+    }
+    
+    /**
+     * Sets values for query.
+     * If versionsToQuery is null or empty, the current version with its 5 preceding versions will be used instead.
+     * 
+     * @param versionsToQuery 
+     * @param startTime
+     * @param endTime
+     */
+    public void setQuery(List<String> versionsToQuery, Long startTime, Long endTime) {
+        query.includeAppLogs(INCLUDE_APP_LOG);
+        query.batchSize(BATCH_SIZE);
+        query.minLogLevel(MIN_LOG_LEVEL);
+        setTimePeriodForQuery(startTime, endTime);
+        setEndTime(endTime);
+        versionList = getVersionIdsForQuery(versionsToQuery);
+        query.majorVersionIds(versionList);
+    }
+    
+    /**
+     * Sets time period to search for query.
+     * @param startTime
+     * @param endTime
+     */
+    private void setTimePeriodForQuery(Long startTime, Long endTime) {
+        if (startTime != null) {
+            query.startTimeMillis(startTime);
+        }
+        if (endTime != null) {
+            query.endTimeMillis(endTime);
+        }
+    }
+    
+    /**
+     * Sets end time of the query.
+     */
+    public void setEndTime(Long endTime) {
+        this.endTime = endTime;
+    }
+    
+    /**
+     * Gets end time of the query.
+     */
+    public Long getEndTime() {
+        return endTime;
+    }
+    
+    /**
+     * Gets versions used in query.
+     */
+    public List<String> getVersionsToQuery() {
+        return versionList;
+    }
+    
+    /**
+     * Selects versions for query. If versions are not specified, it will return 
+     * default versions used for query.
+     */
+    private List<String> getVersionIdsForQuery(List<String> versions) {
+        boolean isVersionSpecifiedInRequest = (versions != null && !versions.isEmpty());
+        if (isVersionSpecifiedInRequest) {
+            return versions;
+        }
+        return getDefaultVersionIdsForQuery();
+    }
+    
+    
+    
+    /**
+     * Gets a list of versions, including the current version and 5 preceding versions (if available).
+     * @return a list of default versions for query.
+     */
+    private List<String> getDefaultVersionIdsForQuery() {
+        List<Version> versionList = Version.getAvailableVersions();
+        Version currentVersion = Version.getCurrentVersion();
+        
+        List<String> defaultVersions = new ArrayList<String>();
+        try {
+            int currentVersionIndex = getCurrentVersionIndex(versionList, currentVersion);
+            defaultVersions = getNextFewVersions(versionList, currentVersionIndex);
+        } catch (InvalidParametersException e) {
+            defaultVersions.add(currentVersion.toStringForQuery());
+            Utils.getLogger().severe("The current version is not found: " + e.getMessage());
+            e.printStackTrace();
```

Comment:
Two issues:
- Don't include `e.printStackTrace()`.
- The message "The current version is not found" is almost identical to what is set in the Exception. `.severe(e.getMessage())` should be sufficient.